eth/downloader/modes.go

```diff
@@ -29,7 +29,7 @@ const (
 )
 
 func (mode SyncMode) IsValid() bool {
-	return mode >= FullSync && mode <= LightSync
+	return mode >= FullSync && mode < LightSync
```

Comment:
We should also get rid of `downloader.LightSync` constant IMO.

Reply:
So we are okay to remove the deprecated light flags as well then?